vcd/resource_vcd_org_vdc.go

```diff
@@ -173,9 +173,10 @@ func resourceVcdOrgVdc() *schema.Resource {
 				Description: "Boolean to request thin provisioning. Request will be honored only if the underlying datastore supports it. Thin provisioning saves storage space by committing it on demand. This allows over-allocation of storage.",
 			},
 			"network_pool_name": &schema.Schema{
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "The name of a network pool in the Provider VDC. Required if this VDC will contain routed or isolated networks.",
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: suppressEmptyString(),
```

Comment:
When does this issue occur? I would assume that if someone is using NSX-T based vDC - he would always get empty value, while if he is not - he would always get a real value. Isn't it so?

Reply:
yes

Reply:
Wouldn't it be safer to `d.Set` the field in `Create` function as it is. That way there is no need for such a "wide capturing" suppress function.

Reply:
On create we set the value, issue is when we re read it/refresh